content/kotlin/concepts/datetime/terms/atDate/atDate.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.atDate()'
+Description: 'Combines a date and time into a LocalDateTime object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.atDate()`** method in Kotlin is a powerful tool that combines a date and time into a `LocalDateTime` object. This method is particularly useful when you need to create a complete timestamp by merging separate `LocalDate` and `LocalTime` instances.
+
+## Syntax
+
+```pseudo
+localTime.atDate(localDate)
+```
+
+- `localTime`: The time component.
+- `localDate`: The date component.
+
+## Example
+
+This example demonstrates combining a time and a date:
+
+```kotlin
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.LocalDateTime
+
+fun main() {
+    val date = LocalDate.of(2023, 12, 25)
+    val time = LocalTime.of(15, 30)
+    val dateTime = time.atDate(date)
+
+    println(dateTime)
+}
+```
+
+The output will be:
```

Comment:
```suggestion
The above example will give the following output:
```

Reply:
Thanks for taking the time to review! I will fix those and sync it.